### PR TITLE
Picekl/geoclip

### PR DESCRIPTION
**Pull request by picekl**
## :memo: Changelog

- [x] GeoCLIP Encoder was added as a pre-trained model.

## :link: Links
- Reference to the [Original Implementation](https://github.com/VicenteVivan/geo-clip)
## :white_check_mark: Checklist

- [x] Lint and tests pass locally with my changes
- [x] I've added the necessary DocString documentation

### CODE REVIEW

**Review by tlarcher — CHANGES_REQUESTED** [review]
location changes

File: malpolon/pretrained/geoclip.py [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+        A tensor of specified size sampled from the normal dist. with mean 0 and variance sigma^2.
+
+    See Also
+    --------
+    rff.layers.GaussianEncoding : For more details on usage.
+    """
+    return torch.randn(size) * sigma
+
+
+@torch.jit.script
```
[the remark this answers is not in the document]
Comment (picekl) [reply malpolon/pretrained/geoclip.py:114]: I would keep it as it is in the official implementation. It should help with optimization.

**Comment by tlarcher** [comment]
The code provided looks fine to me. However before pushing it to dev, I feel like we should add an example of it in `custom_train` (no dataset, or just sample) or `benchmarks` (with dataset) first.
It will be necessary before pushing it to main anyway

**Comment by picekl** [comment]
@tlarcher I agree with the "baseline example". Besides, before the merge, I would like to first finish the SatClip model as well.

**Pull request closed without merging**